views/js/validation.js

```diff
@@ -84,6 +84,23 @@ $(document).ready(function () {
             $('.alert.alert-success').hide();
             showErrorMessage(description_message);
         }
+
+        var minAmount = $paymentMethodForm.find('input[name^="MOLLIE_METHOD_MIN_AMOUNT"]');
+        var maxAmount = $paymentMethodForm.find('input[name^="MOLLIE_METHOD_MAX_AMOUNT"]');
+
+        if (parseFloat(minAmount.val()) < parseFloat(minAmount.attr('min')) || parseFloat(minAmount.val()) > parseFloat(maxAmount.attr('max'))) {
+            event.preventDefault();
+            minAmount.addClass('mollie-input-error');
+            $('.alert.alert-success').hide();
+            showErrorMessage(min_amount_message);
+        }
+
+        if (parseFloat(maxAmount.val()) < parseFloat(minAmount.attr('min')) || parseFloat(maxAmount.val()) > parseFloat(maxAmount.attr('max'))) {
+            event.preventDefault();
+            maxAmount.addClass('mollie-input-error');
+            $('.alert.alert-success').hide();
+            showErrorMessage(max_amount_message);
+        }
```

Comment:
why you duplicated the validation in the front-end as well, what is the use case ? Maybe you can keep everything in back-end and instead do an ajax rqeuest ?

Reply:
in admin page we validate if the amount is correct for the amount in API. in the front we have to convert total price and check if that part is correct.